src/hooks/usePaginatedReportActions.ts

```diff
@@ -0,0 +1,33 @@
+import {useMemo} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import * as ReportActionsUtils from '@libs/ReportActionsUtils';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+/**
+ * Get the longest continuous chunk of reportActions including the linked reportAction. If not linking to a specific action, returns the continuous chunk of newest reportActions.
+ */
+function usePaginatedReportActions(reportID?: string, reportActionID?: string) {
+    // Use `||` instead of `??` to handle empty string.
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
Why do we even have this rule? This is like the 3rd time I've seen it dismissed today.